modules/base/src/net/server.fz

```diff
@@ -107,7 +107,7 @@ is
 
     # bind to port
     (fuzion.sys.net.bind f.as_num s.as_num p.as_num addr port)
-      .bind sd->
+      .bind i32 sd->
```

Comment:
Is this an accidental commit or does your PR worsen the type inference?

Reply:
I created an issue: #5054